imap_processing/tests/spice/test_kernels.py

```diff
@@ -1,33 +1,65 @@
 """Tests coverage for imap_processing/spice/kernels.py"""
 
+import numpy as np
 import pytest
 import spiceypy as spice
 from spiceypy.utils.exceptions import SpiceyError
 
-from imap_processing.spice import kernels
+from imap_processing.spice.kernels import (
+    _average_quaternions,
+    _create_rotation_matrix,
+    _get_et_times,
+    create_pointing_frame,
+    ensure_spice,
+)
+
+
+@pytest.fixture()
+def kernels(spice_test_data_path):
```

Comment:
This fixture seems sensitive to extra kernels being added to the test data path. Each test should have a distinct set of kernels it requires rather than just furnishing everything in the test data path. Perhaps this fixture could be called `pointing_frame_kernels` and return a hard coded list of those kernels?